features/in_onderzoek.feature

```diff
@@ -185,721 +184,938 @@ Functionaliteit: in onderzoek
+      | type                | RaadpleegMetBurgerservicenummer               |
+      | burgerservicenummer | 000000565                                     |
+      | fields              | immigratie.indicatieVestigingVanuitBuitenland |
+      Dan heeft de response een persoon met de volgende 'immigratie' gegevens
+      | naam                                                | waarde       |
+      | inOnderzoek.indicatieVestigingVanuitBuitenland      | true         |
+      | inOnderzoek.datumIngangOnderzoek.datum              | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoek.type               | Datum        |
+      | inOnderzoek.datumIngangOnderzoekPersson.langFormaat | 7 maart 2022 |
+
     Abstract Scenario: verblijfplaats adresregel1, adresregel2 en woonplaats in onderzoek bij <gegeven in onderzoek> is in onderzoek
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550001 |
-      En de persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfplaats.inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      Gegeven de persoon met burgerservicenummer '000000577' heeft de volgende 'verblijfplaats' gegevens
+      | naam                            | waarde   |
+      | aanduiding in onderzoek (83.10) | <waarde> |
+      | datum ingang onderzoek (83.20)  | 20220307 |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                                              |
       | type                | RaadpleegMetBurgerservicenummer                     |
-      | burgerservicenummer | 555550001                                           |
+      | burgerservicenummer | 000000577                                           |
       | fields              | adresregel1,adresregel2,straat,korteNaam,woonplaats |
-      Dan heeft persoon met burgerservicenummer '555550001' de volgende 'verblijfplaats.inOnderzoek' gegevens
-      | naam                             | waarde        |
-      | adresregel1                      | <adresregel1> |
-      | adresregel2                      | <adresregel2> |
-      | korteNaam                        | <korteNaam>   |
-      | straat                           | <straat>      |
-      | woonplaats                       | <woonplaats>  |
-      | datumIngangOnderzoek.datum       | 2022-03-07    |
-      | datumIngangOnderzoek.type        | Datum         |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022  |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                                                       | waarde               |
+      | inOnderzoek.adresregel1                                    | <adresregel1>        |
```

Comment:
inOnderzoek.adresregel1 en inOnderzoek.adresregel2 komen niet voor in 'verblijfplaats' maar in adressering.